integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/ForespoerselMottattIT.kt

```diff
@@ -22,44 +31,156 @@ import java.util.UUID
+                data[Key.PERSONER]
+                    ?.fromJson(personMapSerializer)
+                    .shouldNotBeNull()
+            }
+
+        messagesFilteredForespoerselMottatt
+            .filter(BehovType.OPPRETT_SAK)
+            .firstAsMap()
+            .also {
+                it[Key.UUID]?.fromJson(UuidSerializer).shouldNotBeNull()
+                it[Key.ARBEIDSTAKER_INFORMASJON]?.fromJson(PersonDato.serializer()).shouldNotBeNull()
+
+                it[Key.FORESPOERSEL_ID]?.fromJson(UuidSerializer) shouldBe Mock.forespoerselId
+                it[Key.ORGNRUNDERENHET]?.fromJson(Orgnr.serializer()) shouldBe Mock.orgnr
+            }
+
+        messagesFilteredForespoerselMottatt
+            .filter(BehovType.OPPRETT_OPPGAVE)
+            .firstAsMap()
+            .also {
+                it[Key.UUID]?.fromJson(UuidSerializer).shouldNotBeNull()
+                it[Key.VIRKSOMHET]?.fromJson(String.serializer()).shouldNotBeNull()
+
+                it[Key.FORESPOERSEL_ID]?.fromJson(UuidSerializer) shouldBe Mock.forespoerselId
+                it[Key.ORGNRUNDERENHET]?.fromJson(Orgnr.serializer()) shouldBe Mock.orgnr
             }
 
         messages
-            .filter(EventName.FORESPØRSEL_LAGRET)
+            .filter(EventName.SAK_OG_OPPGAVE_OPPRETT_REQUESTED)
             .firstAsMap()
             .also {
-                it shouldNotContainKey Key.BEHOV
+                it[Key.UUID]?.fromJson(UuidSerializer).shouldNotBeNull()
 
-                Key.EVENT_NAME.lesOrNull(EventName.serializer(), it) shouldBe EventName.FORESPØRSEL_LAGRET
-                Key.FORESPOERSEL_ID.lesOrNull(UuidSerializer, it) shouldBe Mock.forespoerselId
-                Key.ORGNRUNDERENHET.lesOrNull(Orgnr.serializer(), it) shouldBe Mock.orgnr
-                Key.IDENTITETSNUMMER.lesOrNull(Fnr.serializer(), it) shouldBe Mock.fnr
+                val data = it[Key.DATA].shouldNotBeNull().toMap()
+                data[Key.SYKMELDT]?.fromJson(Person.serializer()).shouldNotBeNull()
+                data[Key.VIRKSOMHET]?.fromJson(String.serializer()).shouldNotBeNull()
+
+                data[Key.FORESPOERSEL_ID]?.fromJson(UuidSerializer) shouldBe Mock.forespoerselId
+                data[Key.ORGNRUNDERENHET]?.fromJson(Orgnr.serializer()) shouldBe Mock.orgnr
             }
+
+        messages
+            .filter(EventName.SAK_OPPRETTET)
+            .firstAsMap()
+            .also {
+                val data = it[Key.DATA].shouldNotBeNull().toMap()
+                data[Key.SAK_ID]?.fromJson(String.serializer()) shouldBe Mock.sakId
+            }
+
+        messages
+            .filter(EventName.OPPGAVE_OPPRETTET)
+            .firstAsMap()
+            .also {
+                val data = it[Key.DATA].shouldNotBeNull().toMap()
+                data[Key.OPPGAVE_ID]?.fromJson(String.serializer()) shouldBe Mock.oppgaveId
+            }
+    }
+
+    @Test
+    fun `Slett sak loeser test`() {
```

Comment:
Hører dette til i ForespoerselMottattIT?
Det er kanskje overkill med en ny IT.  Er det derfor den er flyttet hit?

Reply:
> Hører dette til i ForespoerselMottattIT?

Nei, strengt tatt ikke, men ...

> Det er kanskje overkill med en ny IT. Er det derfor den er flyttet hit?
🎯 

Jeg skal uansett fjerne testen i en nært forestående PR.